pkgs/os-specific/linux/ati-drivers/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, kernel ? null, which
+{ stdenv, fetchurl, kernel ? null, linux, which
```

Comment:
I expect you have a 64-bit system with `driSupport32Bit = true`: there you get 64-bit version build against the kernel **and** a 32-bit version with `libsOnly == true` (i.e. without kernel).